binding/form_mapping.go

```diff
@@ -70,20 +70,23 @@ func mapping(value reflect.Value, field reflect.StructField, setter setter, tag
 		return isSetted, nil
 	}
 
-	ok, err := tryToSetValue(value, field, setter, tag)
-	if err != nil {
-		return false, err
-	}
-	if ok {
-		return true, nil
+	if !(vKind == reflect.Struct && field.Anonymous) {
```

Comment:
thanks! I suggest change it to 
`if vKind != reflect.Struct || !field.Anonymous {`

Reply:
Thank you, I have modified it.